packages/nvidia_nat_config_optimizer/src/nat/plugins/config_optimizer/prompts/ga_prompt_optimizer.py

```
@@ -0,0 +1,697 @@
+        eval_metrics = [v.evaluator_name for v in metric_cfg.values()]
+
+        if optimizer_config.output_path is None:
+            raise ValueError("optimizer_config.output_path cannot be None for GA prompt optimization")
+        out_dir = optimizer_config.output_path
+        out_dir.mkdir(parents=True, exist_ok=True)
+
+        prompt_cfg = optimizer_config.prompt
+        pop_size = max(2, int(prompt_cfg.ga_population_size))
+        generations = max(1, int(prompt_cfg.ga_generations))
+        elitism = max(0, int(prompt_cfg.ga_elitism))
+        offspring_size = prompt_cfg.ga_offspring_size or max(0, pop_size - elitism)
```

Ruff (0.15.5)

[error] Local variable `offspring_size` is assigned to but never used

Remove assignment to unused variable `offspring_size`

(F841)